vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -23,12 +23,20 @@ The list of Conferences must be visible to only the `conference-user` and `confe
 ```java
     import org.springframework.security.access.prepost.PreAuthorize;
 ```
-4. Modify the REST API `Conference:getAllConferences` method by preceding it with the @PreAuthorize annotation. Your method signature may be different depending on your blueprint selections.
+4. Modify the REST API `Conference:getAllConferences` method by preceding it with the @PreAuthorize annotation shown here: 
 ```java{1}
     @PreAuthorize("hasAnyAuthority('conference-user','conference-admin')")
-    public List<Conference> getAllConferences()
 ```
-This confines use of the `getAllConferences` method to users who are assigned either the `conference-user` or the `conference-admin` role on the Keycloak client configured for the microservice. 
+This confines the use of the `getAllConferences` method to users who are assigned either the `conference-user` or the `conference-admin` role on the Keycloak client configured for the microservice. Your method signature may be different depending on your blueprint selections, but this an example of the updated section:
+``` java{1}
+ @GetMapping("/conferences")
+    @PreAuthorize("hasAnyAuthority('conference-user','conference-admin')")
+    public ResponseEntity<List<Conference>> getAllConferences(@org.springdoc.api.annotations.ParameterObject Pageable pageable) {
```

Comment:
The signature is just the lines with the @ (annotations) plus the method itself _<visibility> <returnType> <methodName>(<arguments) so you could remove lines 35-38 for simplicity.